### DISCUSSION TITLE
Loading a scene in Jupyter notebook

### DISCUSSION THREAD

**Discussion by ashwinsathish** [NVlabs/sionna#149] [Q&A] [answered]
Dear developers

I have been trying to load a scene using the command - `scene = load_scene(sionna.rt.scene.munich)` but Jupyter throws a runtime error saying:

RuntimeError: ​[xml.cpp:215] Error while loading "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/sionna/rt/scenes/munich/munich.xml" (at line 11, col 3): jit_init_thread_state(): the LLVM backend is inactive because the LLVM shared library ("libLLVM.dylib") could not be found! Set the DRJIT_LIBLLVM_PATH environment variable to specify its path.

Kindly request any assistance that could be provided in this regard.

<img width="1082" alt="Screenshot 2023-06-10 at 12 08 51 AM" src="https://github.com/NVlabs/sionna/assets/94456641/599ad132-3c2b-4d6e-a23f-7f0ee6f7725e">

**Comment by merlinND** [comment, marked as answer]
Hello @ashwinsathish,

Assuming that your machine does not have a CUDA-capable GPU, could you please follow these instructions to enable CPU support (using the LLVM backend):
https://drjit.readthedocs.io/en/latest/firststeps-py.html#llvm-backend

**Comment by ashwinsathish** [comment]
Thank you very much. It works now

On Mon, 12 Jun 2023 at 12:41 PM, Merlin Nimier-David <
***@***.***> wrote:

> Hello @ashwinsathish <https://github.com/ashwinsathish>,
>
> Assuming that your machine does not have a CUDA-capable GPU, could youp
> lease follow these instructions to enable CPU support (using the LLVM
> backend):
> https://drjit.readthedocs.io/en/latest/firststeps-py.html#llvm-backend
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NVlabs/sionna/discussions/149#discussioncomment-6148957>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWQUWQKRP7Y3TMNIXFFMBP3XK26IJANCNFSM6AAAAAAZA7U3IY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
The information contained in this electronic communication is intended 
solely for the individual(s) or entity to which it is addressed. It may 
contain proprietary, confidential and/or legally privileged information. 
Any review, retransmission, dissemination, printing, copying or other use 
of, or taking any action in reliance on the contents of this information by 
person(s) or entities other than the intended recipient is strictly 
prohibited and may be unlawful. If you have received this communication in 
error, please notify us by responding to this email or telephone and 
immediately and permanently delete all copies of this message and any 
attachments from your system(s). The contents of this message do not 
necessarily represent the views or policies of BITS Pilani.
